src/components/PupilJourneyComponents/QuizShortAnswer/QuizShortAnswer.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import {
+  OakBox,
+  OakThemeProvider,
+  oakDefaultTheme,
+} from "@oak-academy/oak-components";
+
+import { QuizShortAnswer } from "./QuizShortAnswer";
+
+import {
+  QuestionFeedbackType,
+  QuestionModeType,
+  QuizEngineContext,
+  QuizEngineContextType,
+} from "@/components/PupilJourneyComponents/QuizEngineProvider";
+import { quizQuestions } from "@/node-lib/curriculum-api-2023/fixtures/quizElements.fixture";
+
+const meta = {
+  component: QuizShortAnswer,
+  tags: ["autodocs"],
+  decorators: [
+    (Story) => (
+      <OakThemeProvider theme={oakDefaultTheme}>
+        <Story />
+      </OakThemeProvider>
+    ),
+  ],
+  parameters: {
+    controls: {
+      include: [],
+    },
+  },
+} satisfies Meta<typeof QuizShortAnswer>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+// mock the QuizEngineContext
+const getContext = (
```

Comment:
I think i've been spoilt by libraries that auto mock through introspection but this looks annoying to have written 😆